packages/fxa-customs-server/lib/dataflow.js

```diff
@@ -75,12 +81,16 @@ module.exports = (config, log, fetchRecords, setRecords) => {
           [type]: action.indicator,
         });
 
-        records[type][action.suggested_action]();
+        if (recordType) {
+          records[recordType][action.suggested_action]();
```

Comment:
this was undefined before because it was doing `records['email'].block()`, it should be `records['emailRecord'].block()`